extensions/apps/antenna/src/components/pages/entry-page/beam-section.tsx

```diff
@@ -25,10 +26,15 @@ const BeamSection: React.FC<BeamSectionProps> = props => {
   const [isReflecting, setIsReflecting] = useState(
     routerState.location.pathname.endsWith(routes[REFLECT]),
   );
+  const activeBeam = beamData?.active;
 
   return (
-    <Stack dataTestId="beam-section" spacing="gap-y-2" customStyle="grow mb-2">
-      <Stack customStyle="grow">
+    <Stack
+      dataTestId="beam-section"
+      spacing="gap-y-2"
+      customStyle={`grow ${activeBeam ? 'mb-2' : ''}`}
+    >
+      <Card customStyle="grow" type="plain">
```

Comment:
any preference for switching to Card type of 'plain' instead of the Stack?

Reply:
yes, there is no need to apply any style except flex grow